xarray/core/dataarray.py

```diff
@@ -1128,11 +1129,11 @@ def from_dict(cls, d):
 
         d = {'dims': ('t'), 'data': x}
 
-        d = {'coords': {'t': {'dims': 't', 'data': t,
-                              'attrs': {'units':'s'}}},
-             'attrs': {'title': 'air temperature'},
-             'dims': 't',
-             'data': x,
+        d = {'coords': {'t': {'dims': 't', 'data': t, \
```

Comment:
I would use `::` and indentation, which will format this as code in the HTML docs: https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt#other-points-to-keep-in-mind